data/human/human missions.txt

```diff
@@ -107,7 +107,269 @@ mission "Pookie, Part 2"
+				goto kindly
+			label turner
+			`	You turn around and come face-to-face with a dour-looking dog looking up at you with puppy-dog eyes. Inspecting it, you recognize the poodle as Puny, who must have snuck out of the complex during all the commotion.`
+			label kindly
+			choice
+				`	(Pet her.)`
+				`	(Shoo her.)`
+					goto shoo
+			`	You crouch down and rub the soggy tufts of hair on Puny's head. As you continue, a smile creeps onto her face, beaming in spite of the soaking rain. You think about what to do with her.`
+			choice
+				`	(Bring Puny aboard your ship.)`
+				`	(Shoo Puny off.)`
+					goto shoo
+			`	You pick up Puny and carry her onboard. As you dry yourselves, you think back to what the announcer said about Puny being owned by someone from the Paradise Worlds, and wonder where you might be able to drop her off.`
+			branch remember
+				not "bet on hound racing"
+			`	As you step into the cockpit, you recall the bet you put on the hound race. Looks like you won't be getting your money back.`
+			label remember
+			``
+				accept
+			label shoo
+			`	You wiggle your leg around to scare Puny off. She runs into the mist, vanishing from sight. You turn back and manage to reach your ship, changing into a dry set of clothes.`
+			branch rememberdecline
+				not "bet on hound racing"
+			`	As you step into the cockpit, you recall the bet you put on the hound race. Looks like you won't be getting your money back.`
+			label rememberdecline
+			``
+				decline
+	to complete
+		never
+
+
+
+mission "Pookie, Part 3a"
+	invisible
+	source "Freedom"
+	destination "Hestia"
+	to offer
+		has "Pookie, Part 2: done"
+	on complete
+		fail "Pookie, Part 3"
+
+
+
+mission "Pookie, Part 4"
+	repeat
+	landing
+	source
+		attributes "paradise"
+		not planet "Hestia"
+	to offer
+		has "Pookie, Part 3: active"
+	on offer
+		conversation
+			branch repeat
+				has "Pookie, Part 4: offered"
+			`After landing on <origin>, you look for a way to reach as many people on the planet as possible in order to find Puny's owner. One of the receptionists at a planetary information desk says that she can put out a notice for a lost dog on a planetwide notice board, if you're able to put in a detailed enough description of Puny. You describe the poodle's appearance to the best of your abilities, and also mention how she was traveling for a vacation when she was abducted.`
+			`	When you check back with the receptionist the next morning, she tells you that nobody has approached the offer.`
+				decline
+			label repeat
+			`You put a description of Puny on a planetwide notice board. By the time the next day arrives, you've received no offers in response.`
+				decline
+
+
+
+mission "Pookie, Part 5"
+	landing
+	source "Hestia"
+	to offer
+		has "Pookie, Part 3a: done"
+	on offer
```

Comment:
If we do this instead then we don't need Part 3a.
```suggestion
		has "Pookie, Part 3: active"
	on offer
		fail "Pookie, Part 3"
```

Reply:
The problem with doing it this way is that it's too slow: Pookie, Part 3 isn't removed from the mission list until taking off/reloading the save. Putting the fail in an on complete block makes it fail properly.

Reply:
So #1938? I'd rather have cleaner mission structures that assume that that issue is eventually fixed that have mission structured to avoid what is essentially a visual bug.